coprocessor/fhevm-engine/transaction-sender/tests/add_ciphertext_tests.rs

```diff
@@ -349,7 +349,12 @@ async fn retry_on_transport_error(#[case] signer_type: SignerType) -> anyhow::Re
         ProviderBuilder::default()
             .filler(FillersWithoutNonceManagement::default())
             .wallet(env.wallet.clone())
-            .connect_ws(WsConnect::new(env.ws_endpoint_url()))
+            .connect_ws(
```

Comment:
Is it really  part of this PR ?

Reply:
No, it isn't, just noticed it taking more time than I would expect. I can put it in a separate PR?